.github/workflows/nightly-ecr.yml

```diff
@@ -47,3 +47,14 @@ jobs:
       - name: Save logs
         if: always()
         uses: ./.github/actions/save-logs
+
+      - name: Send trigger to Slack on workflow failure
+        if: failure()
+        uses: slackapi/slack-github-action@e28cf165c92ffef168d23c5c9000cffc8a25e117 #v1.24.0
+        with:
+          payload: |
+            {
+              "text": "The GitHub Action '${{ github.action }}' had a result of: ${{ job.status }}."
+            }
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
```

Comment:
this is prob better put in `.github/actions` as an action w/ inputs to be more easily consumed